lib/drivers/onetouch/oneTouchVerio.js

```diff
@@ -318,16 +332,17 @@ class BlockDevice {
 
   closeDevice(callback) {
     if (this.fileHandle) {
-      fs.close(this.fileHandle);
-      this.fileHandle = null;
-    }
-    if (callback) {
+      fs.close(this.fileHandle, () => {
+        this.fileHandle = null;
+        return callback();
```

Comment:
I think this should get a check before calling callback, since it may not be defined when called from e.g. checkDevice. (Same as done in the else clause below.)

Reply:
Good catch! I'm also going to replace `fs.close` with `fs.closeSync` to simplify the code a little bit.